pkg/cable/driver.go

```diff
@@ -50,12 +50,12 @@ func AddDriver(name string, driverCreate DriverCreateFunc) {
 }
 
 // Returns a new driver according the required Backend
-func NewDriver(localSubnets []string, localEndpoint types.SubmarinerEndpoint) (Driver, error) {
-	driverCreate, ok := drivers[localEndpoint.Spec.Backend]
+func NewDriver(subnets []string, endpoint types.SubmarinerEndpoint, cluster types.SubmarinerCluster) (Driver, error) {
```

Comment:
nit: can we keep this localSubnets / localEndpoints like in the drivers so they are uniform?
```suggestion
func NewDriver(localSubnets []string, localEndpoint types.SubmarinerEndpoint, cluster types.SubmarinerCluster) (Driver, error) {
```

Reply:
DONE